docs/visaodoproduto/tap.md

```diff
@@ -0,0 +1,149 @@
+
+## Requisitos de alto nível
+Ainda não há dados a respeito.
+
+## Restrições do projeto
+As restrições do presente projeto são:
+
+- Aplicação dependente de conexão à internet para ser utilizada;
+- Não há garantias de resolução da denúncia pelo Estado;
+- A aplicação não faz uma ponte entre o governo e o usuário, mas sim se responsabiliza pela comunicação com o governo a partir da denúncia do usuário.
+
+## Riscos iniciais
+Durante a fase de iniciação do projeto, riscos foram levantados e serão rastreados continuamente, de forma que a equipe de gerência consiga prevenir, transferir, mitigar ou aceitá-los, para que o projeto ocorra sem atrasos ou grandes impedimentos que possam comprometê-lo ou até mesmo inviabilizá-lo.
+
+### Riscos internos
+- Indefinição de escopo
+- Incertezas com relação ao tema do projeto
+- Membro faltante nas reuniões semanais
+- Divergência de horários dos membros
+- Falta de integração entre a equipe
+- Baixa produtividade dos integrantes da equipe
+- Dificuldade com tecnologias escolhidas
+- Requisitos com alto nível de complexidade
+- Falta de tempo de integrantes da equipe
+- Problemas com ambiente de desenvolvimento
+- Falhas de comunicação
+
+### Riscos externos
+- Pouca adesão ao uso do produto criado
+- Não garantia da resposta do governo às denúncias
+- Rejeição pelos órgãos públicos por inconformidade do relatório
+- Enviar relatórios para órgãos não responsáveis pela denúncia em questão
+- Usabilidade baixa e burocratização da aplicação
+
+O Scrum Master será o responsável por monitorar os riscos internos, e facilitar a adoção de medidas que mitiguem esses riscos, assim como o Product Owner será responsável por gerenciar os riscos externos. O controle é feito através das definições previstas na seção Gerenciamento de Riscos que pode ser encontrada [aqui](../metodologia/gerenciaRisco.md).
+
+## Cronograma e marcos
+|Marco|Data|Atividade|
+|--|--|--|
+|Início do projeto|06/02/21|Início da primeira Sprint do projeto|
+|Release 01|21/03/21 - 27/03/21|Primeira entrega de valor do projeto, que aborda os principais artefatos do início de projeto|
+|Release 02|16/05/21 - 22/05/21|Segunda entrega de valor, que aborda as funcionalidades definidas no escopo|
+
+## Custo Estimado
+Os custos estimados são para 15 sprints de projeto, ou seja, 15 semanas, totalizando 3 meses e 3 semanas de projeto.
+
+### Pessoal
+Utilizando como referência o site de busca de empregos **Glassdoor**, verificamos o salário de cada um dos papéis desempenhados pelo time.
+
+|Papel na equipe|Quantidade de membros na função|Transporte e alimentação/dia|Salário/mês|Total|
+|--|--|--|--|--|
+|Desenvolvedor|6|R$0,00|R$2.772,00|R$16.632,00|
+|DevOps|1|R$0,00|R$11.000,00|R$11.000,00|
+|Product Owner|1|R$0,00|R$7.549,00|R$7.549,00|
+|Scrum Master|1|R$0,00|R$7.945,00|R$7.945,00|
+|Arquiteto|1|R$0,00|R$10.000,00|R$10.000,00|
```

Comment:
Eu espero receber isso no início do mês que vem rsrsrs

Reply:
Rsrsrs sonhei com o salário de vocês... kk